elm.json

```diff
@@ -9,13 +9,13 @@
     ],
     "elm-version": "0.19.0 <= v < 0.20.0",
     "dependencies": {
-        "NoRedInk/datetimepicker-legacy": "1.0.1 <= v < 2.0.0",
+        "NoRedInk/datetimepicker-legacy": "1.0.5 <= v < 2.0.0",
```

Comment:
Same here, let's keep the lower bound the same, unless there's a known reason we need to raise it.